### PR TITLE
🎈 13.1.0

### PR DESCRIPTION
**Pull request by mjcarroll**
# 🎈 Release

Preparation for 13.1.0 release.

Comparison to 13.0.1: https://github.com/gazebosim/sdformat/compare/sdformat13_13.0.1...sdf13

Needed by https://github.com/gazebosim/gz-sim/pull/1560

## Checklist
- [x] Asked team if this is a good time for a release
- [x] There are no changes to be ported from the previous major version
- [x] No PRs targeted at this major version are close to getting in
- [x] Bumped minor for new features, patch for bug fixes
- [x] Updated changelog
- [ ] Updated migration guide (as needed)
- [ ] Link to PR updating dependency versions in appropriate repository in [gazebo-release](https://github.com/gazebo-release) (as needed): <LINK>

**Note to maintainers**: Remember to use **Squash-Merge** and edit the commit message to match the pull request summary while retaining `Signed-off-by` messages.

### CODE REVIEW

File: Changelog.md [hunk 1]
```diff
@@ -1,5 +1,25 @@
...
+    * [Pull request #1180](https://github.com/gazebosim/sdformat/pull/1180)
+
+1. Add a configuration option to resolve URIs
+    * [Pull request #1147](https://github.com/gazebosim/sdformat/pull/1147)
+
+1. World: sdfwarns to sdf::Errors when warnings policy set to sdf::EnforcementPolicy::ERR
+    * [Pull request #1131](https://github.com/gazebosim/sdformat/pull/1131)
+
+1. PrintConfig: add sdf::Errors output to API methods
+    * [Pull request #1095](https://github.com/gazebosim/sdformat/pull/1095)
```
Comment (scpeters) [inline Changelog.md:15]: this is #1098 
```suggestion
    * [Pull request #1098](https://github.com/gazebosim/sdformat/pull/1098)
```

**Review by scpeters — COMMENTED** [review]
I see 29 commits in the branch comparison but only 6 PRs in the changelog

**Comment by mjcarroll** [comment]
> I see 29 commits in the branch comparison but only 6 PRs in the changelog

Do you typically list forward ported commits in the latest version of the changelog?  I believe all the others were from merges.

**Comment by scpeters** [comment]
> > I see 29 commits in the branch comparison but only 6 PRs in the changelog
> 
> Do you typically list forward ported commits in the latest version of the changelog? I believe all the others were from merges.

I don't include forward-ported commits in major releases, but I do for minor releases. So the 13.0.0 changelog included only things new since 12.x, but for 13.1 I would include the things thad didn't make it into 13.0

I'll take care of it

**Comment by scpeters** [comment]
> I don't include forward-ported commits in major releases, but I do for minor releases. So the 13.0.0 changelog included only things new since 12.x, but for 13.1 I would include the things thad didn't make it into 13.0
> 
> I'll take care of it

https://github.com/gazebosim/sdformat/pull/1185/commits/6f81b6fd086333ddbd86298ab3384549af07c223

**Pull request merged**